tests/integration/service/test_projects.py

```diff
@@ -1,10 +1,52 @@
+from pathlib import Path
+
 import pytest
 
-from autoarena.error import NotFoundError
+from autoarena.api import api
+from autoarena.error import MigrationError, NotFoundError
 from autoarena.service.project import ProjectService
+from autoarena.store.database import get_available_migrations, MIGRATION_DIRECTORY
 
 
 def test__connect__failed() -> None:
     with pytest.raises(NotFoundError):
         with ProjectService.connect("does-not-exist"):
             ...
+
+
+def test__migration__new(test_data_directory: Path) -> None:
+    project = ProjectService.create_idempotent(api.CreateProjectRequest(name="test__migration__new"))
+    applied = ProjectService._get_applied_migrations(Path(project.filepath))
+    available = get_available_migrations()
+    assert len(available) == len(applied) > 0
```

Comment:
Is this conditional correct?

Reply:
Yes — want to ensure we've applied all migrations, and that the number of migrations applied is greater than zero. This uses magic Python syntax to chain comparisons:

```
>>> 5 > 4 > 3 == 3 > 2 > 1 == 1 > 0
True
```

Reply:
Neat. I did not know that was a thing